jetty-websocket/javax-websocket-common/src/main/java/org/eclipse/jetty/websocket/javax/common/JavaxWebSocketFrameHandler.java

```diff
@@ -545,13 +545,7 @@ public void onClose(Frame frame, Callback callback)
 
     public void onPing(Frame frame, Callback callback)
     {
-        ByteBuffer payload = BufferUtil.EMPTY_BUFFER;
-
-        if (frame.hasPayload())
-        {
-            payload = ByteBuffer.allocate(frame.getPayloadLength());
-            BufferUtil.put(frame.getPayload(), payload);
-        }
+        ByteBuffer payload = BufferUtil.copy(frame.getPayload());
```

Comment:
This is different then the original version.
The original version maintained the ByteBuffer, even if null/empty (it would use BufferUtil.EMPTY_BUFFER)

Reply:
It actually does the same thing as the original version because `frame.getPayload()` will return `BufferUtil.EMPTY_BUFFER` if it has a null payload.